ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_hyperbolic_functions.py

```diff
@@ -56,3 +56,52 @@ def test_numpy_sinh(
         subok=True,
         test_values=False,
     )
+
+# arcsinh
```

Comment:
There is a lint issue for this function, please fix this

Reply:
Hey, @Unevilicorn can you guide me on how can I solve this issue? I'm new to this.
Thanks